public/app/features/backup/components/BackupInventory/BackupInventory.service.ts

```diff
@@ -0,0 +1,35 @@
+import { api } from 'app/percona/shared/helpers/api';
+import { Backup, BackupResponse } from './BackupInventory.types';
+
+const BASE_URL = '/v1/management/backup/Artifacts';
+
+export const BackupInventoryService = {
+  async list(): Promise<Backup[]> {
```

Comment:
Does this return a Promise or just a list of `Backup[]`?

Reply:
It's still a Promise, only I'm doing a mapping to the desired format before returning

Reply:
I think it stops being a Promise after `await` no ? 🤔

Reply:
It would be if `list()` wasn't `async`. An `async` function always returns a Promise :))